chronomagia/chronomagia.py

```diff
@@ -93,7 +93,9 @@ def __init__(self, bot):
         names_to_card = {x.name_clean: x for x in self.card_data}
 
         # Check if the card name starts with the query
-        matches = list(filter(lambda x: x.startswith(query), names_to_card.keys()))
+import re
+exp = re.compile(r'' + query, re.IGNORECASE)
+matches = list(filter(lambda x: exp.search(x), names_to_card.keys()))

```

Comment:
1) Did you test this? Looking at it it doesn't seem like it would even run, based on the indentation.
2) The comment also no longer matches the implementation.
3) Imports should go at the top
4) r'' + query doesn't make sense, why not just 'query'
5) re.IGNORECASE should be passed as a named parameter
6) Regex is probably unnecessary here, you can use 'x in y' as your test
7) You almost certainly want this matcher as a fallback instead of directly replacing the startswith matcher
8) Actually, the matcher I suggested (basically the same as this matcher except cleaner) is already implemented a few lines down. Since this is a more permissive match than the 'startswith' matcher, if the behavior you're currently experiencing is not what you want, you're basically accepting a random match, which is not an improvement.